CHANGELOG.md

```diff
@@ -4,9 +4,14 @@
 
 ## `0.1.2` - 2018-11-02
 
+### Added
+
+- Added Frames Per Second (FPS) and Unity heap usage as metrics sent by `MetricSendSystem.cs`.
```

Comment:
`MetricsSendSystem.cs` now sends metrics for Frames Per Second (FPS) and Unity heap usage

Reply:
updated